azure-pipelines.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           PYTHON_VERSION: '3.12'
           PYTHON_ARCH: 'x64'
           BUILD_TYPE: 'wheel'
-          NUMPY_MIN: '1.26.0'
+          NUMPY_MIN: '2.1.0'
```

Comment:
did you mean to change to `3.13` above? I don't think that's a blocker here though